data/i18n/hydrogen_fr.ts

```diff
@@ -3183,6 +3182,7 @@ Le chemin vers le script et le nom du script doivent être sans espaces.</transl
     </message>
     <message>
         <source>Use &amp;LASH</source>
+        <extracomment>LASH (acronym for LASH Linux Audio Handler) is a protocol allowing for session management in Linux (recalling songs, preferences, and JACK port connections).</extracomment>
```

Comment:
LASH = Linux Audio Session Handler (if I'm not wrong)

Reply:
Your right, I mixed that up. But according to the https://wiki.linuxaudio.org/apps/categories/lash it's a recursive one `LASH Audio Session Handler`

Reply:
That is right : [http://git.savannah.nongnu.org/cgit/lash.git/tree/README].